openfl-tutorials/interactive_api/PyTorch_Market_Re-ID/workspace/tools.py

```diff
@@ -22,26 +27,26 @@ class AverageMeter:
     Code imported from https://github.com/pytorch/examples/blob/master/imagenet/main.py#L247-L262
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize Average Meter."""
         self.reset()
 
-    def reset(self):
+    def reset(self) -> None:
         """Reset values."""
         self.val = 0
         self.avg = 0
         self.sum = 0
         self.count = 0
 
-    def update(self, val, n=1):
+    def update(self, val: Any, n: Any = 1) -> None:
```

Comment:
According to usage, `val` is expected to be a float number, and `n` is expected to be an integer number.